skin.css

```diff
@@ -1,13 +1,50 @@
+/* css-skin-prabesh */
+
+
 .highlight {
     background-color: #ddd;
 }
 
+/* common style for all buttons with class btn */
 button.btn {
+  font-size: 1.09em;
   border-radius: 3px;
+  color: #2A31BB;
+  background: transparent;

```

Comment:
I think would find this a bit surprising. I would usually expect that the button has a background colour, even if that colour is just white.